### PR TITLE
Update e2e test case

### PR DESCRIPTION
**Pull request by wenqiq**
1. Update the test-e2e script
2. Use CRD ClientSet to get CRD resource 
3. Improve some test cases

### CODE REVIEW

File: test/e2e/framework.go [hunk 1]
```diff
@@ -64,6 +64,7 @@ type TestOptions struct {
 	operatorConfigPath  string
 	logsExportOnSuccess bool
 	withIPPool          bool
```
Comment (timdengyun) [inline test/e2e/framework.go:66]: I think can remove this flag `withIPPool`, @zhengxiexie  since we don't have ippool test.

Comment (wenqiq) [reply test/e2e/framework.go]: Thanks for reviewing. Deleted.

**Review by zhengxiexie — APPROVED** [review]
LGTM

**Review by timdengyun — APPROVED** [review]
LGTM

**Pull request merged**